apps/dashboard/src/components/workflow-editor/steps/conditions/edit-step-conditions.tsx

```diff
@@ -0,0 +1,53 @@
+import { RiCloseLine, RiGuideFill } from 'react-icons/ri';
+import { useNavigate } from 'react-router-dom';
+
+import { CompactButton } from '@/components/primitives/button-compact';
+import { StepDrawer } from '@/components/workflow-editor/steps/step-drawer';
+import { useWorkflow } from '@/components/workflow-editor/workflow-provider';
+import { useFeatureFlag } from '@/hooks/use-feature-flag';
+import { FeatureFlagsKeysEnum } from '@novu/shared';
+import { Button } from '@/components/primitives/button';
+
+export const EditStepConditions = () => {
```

Comment:
The empty edit step conditions drawer. In the upcoming PRs I'll add the conditions editor and form and will restructure components in the body.